RecoTracker/FinalTrackSelectors/plugins/trackAlgoPriorityOrder.h

```diff
@@ -1,154 +1 @@
-#ifndef RecoTracker_FinalTrackSelectors_trackAlgoPriorityOrder_h
-#define RecoTracker_FinalTrackSelectors_trackAlgoPriorityOrder_h
-
-#include "DataFormats/TrackReco/interface/TrackBase.h"
-
-#include <array>
-
-/**
- * The trackAlgoPriorityOrder maps an reco::TrackBase::TrackAlgorithm
- * enumerator to its priority in track list merging. The mapping is
- * needed because the order of the enumerators themselves does not, in
- * general, convey useful information.
- */
-namespace impl {
-
-  /**
-   * This array defines the priority order in merging for the
-   * algorithms. The order is ascending, i.e. the first algorithm has
-   * the highest priority etc. The size of the array should be
-   * reco::TrackBase:algoSize (checked below with static_assert), and
-   * each reco::TrackBase::TrackAlgorithm enumerator should be in the
-   * array exactly once (checked below in findIndex() function).
-   */
-  constexpr reco::TrackBase::TrackAlgorithm algoPriorityOrder[] ={
-    reco::TrackBase::undefAlgorithm,
-    reco::TrackBase::ctf,
-    reco::TrackBase::rs,
-    reco::TrackBase::cosmics,
-    reco::TrackBase::initialStep,
-    reco::TrackBase::lowPtTripletStep,
-    reco::TrackBase::pixelPairStep,
-    reco::TrackBase::detachedTripletStep,
```

Comment:
It's worth to add this to the description of the PR and maybe even a few words in the subject, e.g
"also move detachedTripletStep right after initialStep in priority order where it should be"